cmd/tkns/main.go

```diff
@@ -51,16 +52,36 @@ func init() {
 	rand.Seed(time.Now().UnixNano())
 }
 
+type CookiesVar []*selenium.Cookie
+
+func (av *CookiesVar) String() string {
+	return ""
```

Comment:
Probably not necessary for this, but would be nice if this were complete. Would it make sense to product string `foo=bar,faa=baz` etc. for the cookies included in `CookiesVar`?

Reply:
That would let us add default values for the flag (which I don't think we need here but would be nice anyway). I believe that's really all the String() method is used for within the `flags` lib.